broker/src/main/java/io/zeebe/broker/system/configuration/BackpressureCfg.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.broker.system.configuration;
+
+public class BackpressureCfg implements ConfigurationEntry {
+
+  private boolean enabled = true;
+  private boolean useWindowed = true;
+  private String algorithm = "vegas";
+
+  public boolean isEnabled() {
+    return enabled;
+  }
+
+  public BackpressureCfg setEnabled(boolean enabled) {
+    this.enabled = enabled;
+    return this;
+  }
+
+  public boolean useWindowed() {
+    return useWindowed;
+  }
+
+  public BackpressureCfg setUseWindowed(boolean useWindowed) {
+    this.useWindowed = useWindowed;
+    return this;
+  }
+
+  public LimitAlgorithm getAlgorithm() {
+    try {
+      return LimitAlgorithm.valueOf(algorithm.toUpperCase());
+    } catch (IllegalArgumentException e) {
+      return LimitAlgorithm.VEGAS;
```

Comment:
Hm, is it better to explode if misconfigured instead of picking another algo? As user, what would you expect?

Side note, if we do decide to pick a default, then we should at least log it and tell the user :sweat_smile: But as a user myself I would prefer to fail early if I misconfigured something.

Reply:
Agree. Let it explode :boom: